test-app/app/controllers/editable-node.ts

```diff
@@ -82,6 +82,18 @@ export default class EditableBlockController extends Controller {
   AttributeEditor = AttributeEditor;
   RdfaEditor = RdfaEditor;
 
+  propertyPredicates = [
+    'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',
+    'http://www.w3.org/ns/prov#value',
+  ];
+
+  propertyObjects = [
+    'http://data.vlaanderen.be/ns/besluit#behandelingVanAgendapunt',
```

Comment:
types are PascalCase, it's gonna be confusing to have this wrong in the test-app:
http://data.vlaanderen.be/ns/besluit#BehandelingVanAgendapunt
http://data.vlaanderen.be/ns/besluit#Agendapunt

Reply:
Yeah sorry, I didn't pay too much attention to it, because I thought you were going to fill the array later